src/main/java/com/codegym/jrugotom5/controller/AdvertController.java

```diff
@@ -27,4 +29,9 @@ public List<AdvertBasicInfoDTO> getAllAdverts() {
     public List<AdvertDTO> getAdverts(@RequestParam LocalDate from, LocalDate to) {
         return advertService.getAdvertsByDateRange(from, to);
     }
-}
\ No newline at end of file
+  
+    @GetMapping("/{userId}")
```

Comment:
I think it's bad idea to use such mapping. For example request by URL: localhost:8080/api/123 isn't clear.

But i suggest this controller should be modified.
@RequestMapping("/api") should be changed to @RequestMapping("/api/adverts")
@Volodymyr95 what do you think about it?

Reply:
i think this endpoint belongs to UserController and we should call it like host/api/users/{id}/adverts.

Reply:
> But i suggest this controller should be modified. @RequestMapping("/api") should be changed to @RequestMapping("/api/adverts") @Volodymyr95 what do you think about it?

did this in #8